### DISCUSSION TITLE
请问clash如何排除延迟测速？

### DISCUSSION THREAD

**Discussion by hubumulity** [XIU2/CloudflareSpeedTest#430] [提问]
我在服务器上启动了定时任务脚本，同时启动了clash客户端，由于bat脚本无法控制clash的开关，所以想问下如何在clash规则中排除CloudflareSpeedTest的延迟测速，走代理显示所有节点都可用，这显然是有问题的。

**Comment by hubumulity** [comment]
curl -X PATCH -H "Content-Type: application/json" -d @tunOff.json http://127.0.0.1:4321/configs

**Comment by hubumulity** [comment]
{
"tun": {
        "enable": false,
        "device": "Meta",
        "stack": "gVisor",
        "dns-hijack": [
            "any:53"
        ],
        "auto-route": true,
        "auto-detect-interface": true,
        "inet4-address": [
            "198.18.0.1/30"
        ],
        "file-descriptor": 0
    }
}

**Comment by wutea7** [comment]
```yaml
- PROCESS-NAME,CloudflareST.exe,DIRECT
```
